src/basic/init/basic_cold_start_internal.s

```diff
@@ -44,5 +61,8 @@ basic_cold_start_internal:
 	; Initialize other variables by performing NEW
 	jsr do_new
 
+    ; Copy CHRGET to the zero page
+    jsr copy_CHRGET
```

Comment:
Indentation does not match the rest of the file.

Reply:
My bad. I'll fix